client/src/App.css

```diff
@@ -11,5 +11,4 @@
   text-align: center;
   width: 100vw;
```

Comment:
Small note that I realized: setting 100vw will lead to horizonal scrollbars if there is ever a vertical scrollbar because VW includes the space taken up by the scrollbar. So its better for widths to be left at 100% even at the top level